plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -49,12 +50,14 @@ import {
   IconButton,
   makeStyles,
 } from '@material-ui/core';
+import AddIcon from '@material-ui/icons/Add';
```

Comment:
seems like [this](https://github.com/backstage/backstage/blob/661734ae7519e683b637ed066dd9bad2404d179f/packages/app/src/components/Root/Root.tsx#L24) is the one we use by default in the sidebar